storage/store_pool.go

```diff
@@ -39,15 +42,24 @@ const (
 	// TestTimeUntilStoreDeadOff is the test value for TimeUntilStoreDead that
 	// prevents the store pool from marking stores as dead.
 	TestTimeUntilStoreDeadOff = 24 * time.Hour
+
+	// timeoutForDeclinedReservations is the amount of time to not consider the
+	// store available for up-replication after a reservation was declined.
+	timeoutForDeclinedReservations = 10 * time.Second
```

Comment:
Can we set this to 1 second? Or lower? I know this hasn't been tuned, but I'm not clear what benefit it is providing. Reservation RPCs will not be overloading the receiving node in any current setup. But this setting will add another layer of throttling to rebalancing which is potentially harmful to allocation throughput.